packages/core-data/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+## 2.3.1 (2019-05-21)
+
+### New Feature
+
+- The `deleteEntityRecord` and `removeItems` actions have been added.
+- Entities now support a `forceDelete` attribute to allow the API to delete things that do not go to trash.
```

Comment:
Why isn't this an option of `deleteEntityRecord`? `deleteEntityRecord` should accept query parameters if this is needed because otherwise, we are tying the entity abstraction to the specifics of the WordPress REST API. `?force=true` might not be the way this is done for all entities.

Reply:
Hm good point. To be honest it's hard to tell when you are tying to the WP api and when you're doing normal WordPress way things :)